### PR TITLE
mutation comparison: allow users to filter by mutation type substitution/deletion

### LINKED ISSUE #109: Allow users to filter by mutation type (substitution/deletion)

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #109

### Screenshot

grafik

### PR Checklist

- [x] The implemented feature is covered by an appropriate test.

### CODE REVIEW

File: components/tests/snapshots.spec.ts-snapshots/Mutation-comparison-Story-visualization-mutation-comparison--default-should-match-screenshot-1-firefox-linux.png
Comment (fengelniederhammer) [inline components/tests/snapshots.spec.ts-snapshots/Mutation-comparison-Story-visualization-mutation-comparison--default-should-match-screenshot-1-firefox-linux.png:1]: The Firefox version that Playwright uses doesn't support `text-wrap: nowrap;` - in my normal Firefox, it looks as expected.

File: components/src/preact/mutationComparison/fetchMutationData.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { queryMutations } from '../../query/queryMutations';
```
Comment (JonasKellerer) [inline components/src/preact/mutationComparison/fetchMutationData.ts:1]: Should we also put the other queries (eg. queryPrevalenceOverTime, ...) in the folder of their component? But maybe also in another PR.

Comment (JonasKellerer) [reply components/src/preact/mutationComparison/fetchMutationData.ts:1]: And the naming is there query{...} and not fetch. (But I am fine with either of them)

**Review by JonasKellerer — APPROVED** [review]
LGTM

File: components/src/preact/mutationComparison/fetchMutationData.ts (hunk 1 above)
Comment (fengelniederhammer) [reply components/src/preact/mutationComparison/fetchMutationData.ts]: I renamed it to query..., but I'll lave reorganizing more to a separate PR.

**Pull request merged**